actgen/utils.py

```diff
@@ -4,12 +4,104 @@
 import logging
 from pydoc import locate
 
+import gym
+import argparse
 import torch
-from matplotlib import pyplot as plt
+
+from . import wrappers as wrap
 
 Experience = namedtuple('Experience', ['state', 'action', 'reward', 'next_state', 'done'])
 
 
+class Trial:
+    """
+    a base trial class for running experiments
+    """
+    def __init__(self):
+        pass
+
+    def parse_common_args(self):
```

Comment:
This returns a parser, but it doesn't technically parse anything, so the function name is a bit confusing. I'd prefer something like `get_common_arg_parser()`.